src/System.Private.CoreLib/shared/System/String.Manipulation.cs

```diff
@@ -40,7 +40,7 @@ public static string Concat(object? arg0)
             {
                 return string.Empty;
             }
-            return arg0.ToString();
+            return arg0?.ToString() ?? string.Empty;
```

Comment:
The code checks for `arg0 == null` above. You do not need to check again.

Reply:
Yes, that was my bad, actually I just converted it to a 1 liner.